site/content/docs/cicd/deployment-settings-file/overview.md

```diff
@@ -0,0 +1,53 @@
+# Creating a deployment settings file
+
+A deployment settings file allows you to define the deployment settings of your application in a _JSON_ format. This JSON file can be version controlled and plugged into your CI/CD system for future deployments. 
+
+Deployment settings files could be used as a way to automate your deployments or use them in a [CI/CD setting](../cicd.md) where you would define all the settings that you need to apply for your deployment and then use the `--apply` flag on the CLI to link to the deployment setting file. 
+
+By doing this, the AWS .NET deployment tool reads all the settings you have defined and applies them to the deployment. You will need to do a final confirmation to initiate the deployment in the CLI. However, you can bypass the confirmation by making the deployment a silent one. This can be done by adding the `--silent` flag which will turn off any prompts.
+
+The deployment settings file has the following structure:
+
+```json
+{
+      "AWSProfile": <AWS_PROFILE>,
+      "AWSRegion": <AWS_REGION>,
+      "ApplicationName": <APPLICATION_NAME>,
+      "RecipeId": <RECIPE_ID>,
+      "Settings": <JSON_BLOB>
+}
+```
+
+* _**AWSProfile**_: The name of the AWS profile that was used during deployment.
+
+* _**AWSRegion**_: The name of the AWS region where the deployed application is hosted.
+
+* _**ApplicationName**_: The name that is used to identify your cloud application within AWS. If the application is deployed via AWS CDK, then this name points to the CloudFormation stack.
+
+* _**RecipeId**_: The recipe identifier that was used to deploy your application to AWS.
+
+* _**Settings**_: This is a JSON blob that stores the values of all available settings that can be tweaked to adjust the deployment configuration. This is represented as a _JSON_ object that contains the deployment setting IDs and values as a key/pair configuration. 
+
+If you head over to each individual recipe's documentation page, you will find the recipe specific information that you need to be able to create and fill out the template provided above.
```

Comment:
This sentence sounds too conversationally and not reference documentation. What about something like the following?

```
Each recipe has its own set of settings that can be configured. The following pages in this section list the settings for each recipe that can be used to fill in the `Settings` section of the file.
```

Reply:
Done